Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -342,7 +346,7 @@ public void TestPutDirectoryMixedWildcard()
         [Test]
         [Parallelizable(ParallelScope.All)]
         public void TestPutGetCommand(
-            [Values("gzip", "bzip2", "brotli", "deflate", "raw_deflate", "zstd")] string compressionType, 
+            [Values("none", "gzip", "bzip2", "brotli", "deflate", "raw_deflate", "zstd")] string compressionType,
```

Comment:
is it input/output file compression? please rename accordingly the variable

Reply:
relpaced by sourceFileCompressionType